composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "require": {
         "php": ">=7.4",
-        "spryker-eco/unzer-api": "^1.0.0",
+        "spryker-eco/unzer-api": "dev-feature/cc-16934/dev-unzer-glue-support",
```

Comment:
* Don't forget to change the version to "^1.1.0", it will not happen automatically.

Reply:
Yep, will be done before release. Thanks for short reminder here.

Reply:
Fixed.